forms-flow-web/src/components/FOI/customComponents/Records/index.js

```diff
@@ -1057,63 +1405,76 @@ export const RecordsLog = ({
-              //       {/* <CircularProgress/> */}
-              //       record.isredactionready ?
-              //       <FontAwesomeIcon icon={faCheckCircle} size='2x' color='#1B8103' className={classes.statusIcons}/>:
-              // record.failed ?
-              // <FontAwesomeIcon icon={faExclamationCircle} size='2x' color='#A0192F' className={classes.statusIcons}/>:
-              // <FontAwesomeIcon icon={faSpinner} size='2x' color='#FAA915' className={classes.statusIcons}/>
-              //       </InputAdornment>
-              //   }}
-                InputLabelProps={{ shrink: false }}
-                select
-                name="download"
-                value={currentDownload}
-                onChange={handleDownloadChange}
-                placeholder="Download"
-                variant="outlined"
-                size="small"
-                fullWidth
-              >
-                {recordsDownloadList.map((item, index) => {
-
-                  if (item.id !=0) {
-                    return (
+                  className="download-dropdown custom-select-wrapper foi-download-button"
+                  id="download"
+                  label={currentDownload === 0 ? "Download" : ""}
+                  inputProps={{ "aria-labelledby": "download-label" }}
+                  //   InputProps={{
+                  //     startAdornment: isDownloadInProgress && <InputAdornment position="start">
+                  //       {/* <CircularProgress class="download-progress-adornment"/> */}
+                  //       {/* <CircularProgress/> */}
+                  //       record.isredactionready ?
+                  //       <FontAwesomeIcon icon={faCheckCircle} size='2x' color='#1B8103' className={classes.statusIcons}/>:
+                  // record.failed ?
+                  // <FontAwesomeIcon icon={faExclamationCircle} size='2x' color='#A0192F' className={classes.statusIcons}/>:
+                  // <FontAwesomeIcon icon={faSpinner} size='2x' color='#FAA915' className={classes.statusIcons}/>
+                  //       </InputAdornment>
+                  //   }}
+                  InputLabelProps={{ shrink: false }}
+                  select
+                  name="download"
+                  value={currentDownload}
+                  onChange={handleDownloadChange}
+                  placeholder="Download"
+                  variant="outlined"
+                  size="small"
+                  fullWidth
+                >
+                  {recordsDownloadList.map((item, index) => {
+                    if (item.id != 0) {
+                      return (
                         <MenuItem
                           className="download-menu-item"
                           key={item.id}
                           value={index}
-                          disabled={item.disabled || conversionFormats?.length < 1}
-                          sx={{ display: 'flex' }}
+                          disabled={item.disabled}
```

Comment:
removed "conversionFormats?.length < 1" check as this is not needed for the download.